go_pprof_scraper/manifest.json

```
@@ -0,0 +1,46 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "2b13f6b1-d3ba-4254-93c0-2ceaf783cd85",
+  "app_id": "go-pprof-scraper",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "",
+    "title": "Go-pprof-scraper",
```

```suggestion
    "title": "Go pprof scraper",
```

That's what people will see in the DD UI